app/v2/controllers/ErrorTranslator.scala

```diff
@@ -63,11 +68,13 @@ trait ErrorTranslator {
   implicit val persistenceErrorConverter = new Converter[PersistenceError] {
 
     def convert(persistenceError: PersistenceError): PresentationError = persistenceError match {
-      case PersistenceError.DepartureNotFound(departureId) =>
+      case DepartureNotFound(departureId) =>
         PresentationError.notFoundError(s"Departure movement with ID ${departureId.value} was not found")
-      case PersistenceError.MessageNotFound(movement, message) =>
+      case ArrivalNotFound(arrivalId) =>
+        PresentationError.notFoundError(s"Arrival movement with ID ${arrivalId.value} was not found")
+      case MessageNotFound(movement, message) =>
         PresentationError.notFoundError(s"Message with ID ${message.value} for movement ${movement.value} was not found")
-      case PersistenceError.DeparturesNotFound(eori) =>
+      case DeparturesNotFound(eori) =>
         PresentationError.notFoundError(s"Departure movement IDs for ${eori.value} were not found")
       case PersistenceError.ArrivalsNotFound(eori) =>
```

Comment:
if you're going to remove the `PersistenceError` qualifier, you should remove it from all the messages.

Reply:
done